### DISCUSSION TITLE
Is popper removed from bootstrap.bundle.min.js

### DISCUSSION THREAD

**Discussion by krishnaTORQUE** [twbs/bootstrap#32736] [General] [answered]
Is popper removed from `bootstrap.bundle.min.js` ?

I was using `4.5.2` & it was working fine after upgrade to `4.5.3` its asking to install `popper.js` separately.

However I installed `@popperjs/core` not `popper.js` & still complaining.

I am using `React js`

The Error

```
Failed to compile.

./node_modules/bootstrap/dist/js/bootstrap.js
Cannot find module: 'popper.js'. Make sure this package is installed.

You can install this package by running: npm install popper.js.
```

As you can see `bootstrap.js` require `popper` but I am importing the bundle file.

**Comment by XhmikosR** [comment]
The bootstrap.bundle.js file does include Popper. You seem to include bootstrap.js which does not include Popper.

**Comment by krishnaTORQUE** [comment]
No sir

as I mention above `bootstrap.bundle.min.js` is added & it was working fine before update.

**Comment by XhmikosR** [comment, marked as answer]
Ah, wait. You are not supposed to use `@popperjs/core`. Bootstrap 4.x needs popper.js 1.x so you need to install that.

**Reply by Novizh** [reply]
Hold on, so I was supposed to install the older version of `popper.js` in order for my popovers and tooltips to work?

**Comment by krishnaTORQUE** [comment]
yes this is what i have done after update.

my question is so they have indeed removed popper from bundle?

**Comment by XhmikosR** [comment]
`@popperjs/core` is v2.x which does not work with Bootstrap 4.x as I said above. Anyway, just stick to popper.js for Bootstrap 4.x.
